### ISSUE TITLE
新手入门，请教样本命名的问题。

### ISSUE THREAD

**Issue by SlovEnt** [kerlomz/captcha_trainer#64]
各位老大好，请问我用“东中了投_b5272ec10b055e5889bd63c4d7042a68.png”这种方式命名了文件，按下列配置设置了项目参数：
category: customized ["东", "蒙", "古", "内", "南", "了", "投", "服", "徽", "天", "安", "区", "片", "外", "津", "股", "海", "心", "湖", "来", "中"]
界面参数：label Num：4，channel：1。
配置文件：
Label:
  LabelFrom: FileName
  ExtractRegex: .*?(?=_)
  LabelSplit: null

点击 start training 以后，会卡在 INFO:tensorflow:Start training... 这个位置，请问大佬们，是我配置有问题吗？

Instructions for updating:
Use eager execution and: 
`tf.data.TFRecordDataset(path)`
INFO:tensorflow:Loading Validation DataSet...
INFO:tensorflow:Total 5083 Trains DataSets
INFO:tensorflow:Total 19 Validation DataSets
WARNING:tensorflow:From D:\My_Workbench\Python\captcha_trainer\core.py:139: The name tf.train.get_or_create_global_step is deprecated. Please use tf.compat.v1.train.get_or_create_global_step instead.

WARNING:tensorflow:From D:\My_Workbench\Python\captcha_trainer\loss.py:34: The name tf.nn.ctc_loss_v2 is deprecated. Please use tf.nn.ctc_loss instead.

WARNING:tensorflow:From D:\My_Workbench\Python\captcha_trainer\core.py:206: The name tf.GraphKeys is deprecated. Please use tf.compat.v1.GraphKeys instead.

WARNING:tensorflow:From D:\My_Workbench\Python\captcha_trainer\trains.py:200: The name tf.global_variables_initializer is deprecated. Please use tf.compat.v1.global_variables_initializer instead.

WARNING:tensorflow:From D:\My_Workbench\Python\captcha_trainer\trains.py:202: The name tf.train.Saver is deprecated. Please use tf.compat.v1.train.Saver instead.

WARNING:tensorflow:From D:\My_Workbench\Python\captcha_trainer\trains.py:202: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

': Expected string but found: 'None'
WARNING:tensorflow:./projects/gdll-CNNX-GRU-H64-CTC-C1\model\checkpoint: Checkpoint ignored
INFO:tensorflow:Start training...
